doc/tags.yml

```diff
@@ -335,7 +303,6 @@ mapping_topics:
   - nrf/libraries/others/esb.html: ["esb"]
   - nrf/libraries/others/fem_al.html: ["fem", "nrf21540"]
   - nrf/libraries/others/st25r3911b_nfc.html: ["nfc"]
-  - nrf/libraries/zigbee/index.html: ["zigbee"]
   - nrf/libraries/zigbee/*.html: ["zigbee"]
   - nrf/external_comp/nrf_cloud.html: ["nrf-cloud"]
   - nrf/external_comp/avsystem.html: ["lte"]
```

Comment:
For nrf/releases_and_maturity/software_maturity.html: ["sidewalk", "ble", "blemesh", "matter",
                                                       "thread", "zigbee", "wifi", "lte"] below, dect nr+ could be added.